_drafts/randy/2022-07-17-10-second-video.md

```diff
@@ -0,0 +1,155 @@
+
+We hope to be different than the rest, and that should help visibility out on the sea of hopefuls. 
+
+**IT'S BEEN SUCH A LONG TIME (I THINK I SHOULD BE GOING)**
+
+Now to the part where making video can take a long time, and if somebody is paying - more money. If we are shooting a $250k per second [big name commercial](https://www.therichest.com/the-biggest/the-10-most-expensive-tv-commercials-ever-made), there is endless money and enough time for perfection. 
+
+My perspective is a lower level concern. Sitting here by the light of my dark mode screen, why does it take ME so long to make 10 seconds of video? 
+
+If we needed to make a report, we might reverse engineer what it takes to get the job done: 
+
+     RESOURCES
+          Existing computers, software and 
+          cameras provide quality product
+     BUDGETS
+          none - deal with it.
+     CREATIVE PRIORITIES
+          1) be relevant to the article content
+          2) be flashy and to the point
+          3) subtly demonstrate skills learned
+             in the process
+     PROJECT TIMELINE
+          Yesterday or ASAP, whichever is first
+
+While we stress over 10 seconds, a real film or video production would require many days and weeks to prepare for and manage all of the following subtopics: 
+
+     FINANCING
+     PLANNING
+     HIRING
+          DIRECTOR
+          PRODUCER
+          CAMERA OPERATOR
+          SOUND MIXER
+          GAFFER (LIGHTING)
+     LOCATIONS
+     TALENT
+
+As none of these traditions are needed for our project, we can save weeks of time and loads of cash. Thank goodness for consumer grade technology. 
+
+Wait - some PLANNING and TALENT may still be necessary! 
+
+**CONCEPT IN PROGRESS**
+
+Ideas for creative concepts develop while writing the words to Analog Samplings. In the same way articles change heading and direction mid-draft, it helps to be flexible on video execution. 
+
+Concepts turn into creative choices with the help of STORYBOARDS - a board of stories you are supposing to tell. Professionally this is done on formatted templates with typed words and example pictures and diagrams. 
+
+If there is no team of people to yell at or otherwise communicate with, formality of the planning process need not strictly be enforced. Some decisions are made for you based on assets available, time left on the clock, and random thoughts or feelings in the moment. 
+
+For a scant 10-second video, you likely do not need a spreadsheet to keep track of your project - your head may have capacity for a few items of note (minimum one required). Sometimes you can get away with using the flipbook in your mind. 
+
+Based on the storyboard, some sheep (ewe) has to put together a shot list of media parts needed to pretend together for the show. In the case of the [Car Trek Technology](https://devtails.xyz/@randy/car-trek-technology_blank) video: 
```

Comment:
I think we should embed that specific video here.  A good 80% of people probably won't click through to a different page.  The same:

```
<video width="720" height="auto" controls style="max-width: 100%">
   <source src="/assets/car-trek-technology-720.mp4" type="video/mp4">
</video>
```

Should work here.

Reply:
done